pkg/commands/edit/add/flagsandargs_test.go

```diff
@@ -102,3 +102,33 @@ func TestExpandFileSource(t *testing.T) {
 		t.Fatalf("FileSources is not correctly expanded: %v", fa.FileSources)
 	}
 }
+
+func TestExpandFileSourceWithKey(t *testing.T) {
+	fakeFS := fs.MakeFakeFS()
+	fakeFS.Create("dir/fa1")
+	fakeFS.Create("dir/reademe")
```

Comment:
readme , or reamde if you like neal stephenson

i guess this typo present in earlier test

Reply:
updated